ic-http-mock/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "ic-http-mock"
+description = "Mocking HTTPS Outcalls on the Internet Computer"
+version = "0.1.0"
+edition = "2021"
+authors = ["Maksym Arutyunyan"]
+repository = "https://github.com/TODO/ic-http-mock" # TODO: Update this
+homepage = "https://github.com/TODO/ic-http-mock"   # TODO: Update this
```

Comment:
Will these be updated before this PR is merged? If not, I'd remove them for now.

Reply:
Done.